src/ralph/filters.py

```diff
@@ -1,22 +1,24 @@
 """
 Ralph tracking logs filters.
 """
+
 from .exceptions import EventKeyError
 
 
-def anonymous(events):
-    """Remove anonymous events.
+def anonymous(event):
+    """Remove anonymous event.
 
     Args:
-        events (DataFrame): events to filter
+        event (dict): event to filter
 
     Returns:
-        Filtered pandas DataFrame.
+        event (dict): when event is not anonymous
+        None: otherwise
```

Comment:
As we removed the pandas dependency some changes had to be made for the anonymous filter, is this ok?

Reply:
yes